### DISCUSSION TITLE
custom data layer name in tile set editor

### DISCUSSION THREAD

**Discussion by jonathandw743** [godotengine/godot-proposals#8980] [Editor]
Could the TileSet editor contain the name of the custom data layer instead of or as well as the index? I think the main way people think about custom data layers is by name instead of index as that is how it is presented in the TileSet resource editor.

Resource editor uses the name:
![image](https://github.com/godotengine/godot-proposals/assets/85790952/b955499f-735d-4e89-93af-56d202daf870)

TileSet editor uses the index:
![image](https://github.com/godotengine/godot-proposals/assets/85790952/51064e5c-4724-4158-bd8c-79149d9e71d9)

**Comment by AThousandShips** [comment]
This is a bug, tracked in:
* https://github.com/godotengine/godot/issues/76733

Closing as a duplicate, thank you for reporting
